src/en/index.html

```diff
@@ -337,12 +337,11 @@ <h1>Pikachu Volleyball - Super AI Edition</h1>
 					<p class="reverse-engineered-by">
 						Based on the <a href="https://gorisanson.github.io/pikachu-volleyball/en/" target="_blank">web
 							version</a>
-						developed by <a href="https://gorisanson.github.io/" target="_blank">Lee,
-							Kyutae</a>
+						developed by <a href="https://gorisanson.github.io/" target="_blank">Kyutae Lee</a>
```

Comment:
In the past I had written my name as "Lee, Kyutae". But, nowadays I'm writing my name as "Kyutae Lee" :)